Sage/carbon
# fix(menuitem): check for null children

Fixes: #3540

### Proposed behaviour
`MenuItem` should allow null children. 

### Current behaviour
`MenuItem` currently crashes when it has a null child with the following error: 

![image](https://user-images.githubusercontent.com/14963680/103689237-f0dc9080-4f8a-11eb-8c84-ef7b9dd81c94.png)

### Checklist


### Additional context

### Testing instructions
https://codesandbox.io/s/agitated-shape-zpz7h?file=/src/index.js

See the PR build in the codesandboxbot comment below for the version with these changes in
